File: C/1013 - O maior.c
#include <stdio.h>
#include <stdlib.h>

int main(){

	int a, b, c;
	int maior_ab, maior_abc;

	scanf("%i %i %i", &a, &b, &c);

	maior_ab = (a+b+abs(a-b))/2;
	maior_abc = (maior_ab+c+abs(maior_ab-c))/2;

	printf("%i eh o maior\n", maior_abc);

	return 0;
}

File: Go/1013 - O maior.go
package main

import "fmt"
import "math"

func main(){

	var maior_ab, maior_abc float64
	var a, b, c float64

	fmt.Scan(&a, &b, &c)

	maior_ab = (a+b+math.Abs(a-b))/2
	maior_abc = (maior_ab+c+math.Abs(maior_ab-c))/2

	fmt.Printf("%.0f eh o maior\n", maior_abc)
}